GUI.py

```diff
@@ -76,6 +76,9 @@ def play(self):
         while output != b'' and self.playing:
             out_stream.write(output)
             output = audio_file.readframes(self.CHUNK)
+        self.playing = False
+        self.playButton['text'] = 'Play'
+        print('audio ended')
```

Comment:
It is a good idea to set the text back to play so the end user understands the state of the application. Also printing 'audio ended' is good for indication that the code is operating properly

Reply:
Glad you agree!